packages/devtools_app/lib/src/framework/landing_screen.dart

```diff
@@ -251,20 +251,21 @@ class ImportFileInstructions extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
```

Comment:
@kenzieschmoll I love this lint warning about duplicate code. It false positives a little bit with enum entries and methods with side effects but in general is tremendously useful in helping people optimize duplicate lookups in build methods.

Reply:
> It false positives a little bit with enum entries 

Could you share an example? I'd like to fix that. Not sure that the problem with side effects is fixable, probably a candidate for // ignore.

Reply:
The lint fires for no good reason if you import a library with a named prefix.
For example:
```
import 'library_with_enum' as libraryName
void main() {
  print(libraryName.EnumName.entry1);
  print(libraryName.EnumName.entry2);

}
```

Reply:
Thank you, I've added to backlog!